includes/Discovery.php

```diff
@@ -101,23 +101,26 @@ public function register(): void {
 	/**
 	 * Prints document title for stories.
 	 *
-	 * Works both for classic themes and block themes without any conditionals.
+	 * Works both for classic themes and block themes.
```

Comment:
Should this comment be updated.

Reply:
I updated it already, do you have a recommendation to further update/clarify it?